repo/packages/K/kafka/500/resource.json

```diff
@@ -0,0 +1,62 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.12.0.tar.gz",
+      "bootstrap-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.55.2/bootstrap.zip",
+      "kafka-tgz": "https://downloads.mesosphere.com/kafka/assets/kafka_2.12-1.1.1.tgz",
+      "kafka-jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u192-linux-x64.tar.gz",
+      "kafka-scheduler-zip": "https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/kafka-scheduler.zip",
+      "kafka-statsd-jar": "http://downloads.mesosphere.com/kafka/assets/kafka-statsd-metrics2-0.5.3.jar",
+      "statsd-client-jar": "http://downloads.mesosphere.com/kafka/assets/java-dogstatsd-client-2.3.jar",
+      "setup-helper-zip": "https://downloads.mesosphere.com/kafka/assets/2.4.0-1.1.1/setup-helper.zip",
+      "zookeeper-client-jar": "http://downloads.mesosphere.com/kafka/assets/zookeeper-3.4.10.1.jar",
```

Comment:
> `zookeeper-3.4.10.1.jar`

Shouldn't this be `zookeeper-3.4.13.jar`?

Reply:
Let me double check with @wavesoft

Reply:
Indeed, we didn't notice this. However if we chance this client version we are going to need another week of soaking. 

The API compatibility is not a problem. This client version will be compatible with future and past versions of ZK. However there are indeed some helpful updates in the client jar that could help us (eg. https://issues.apache.org/jira/browse/ZOOKEEPER-2184 ).

Therefore since this is not a blocker, I am proposing to release this ASAP and right after we release a minor revision bump with the new client jar. What do you think @vishnu2kmohan ?

Reply:
@wavesoft Yes, its precisely because of [ZOOKEEPER-2184](https://issues.apache.org/jira/browse/ZOOKEEPER-2184) that I requested that bump :)

We can wait to bump it in the next release after we've added Secure Remote JMX for both Kafka 1.1.1 (and 2.1.1)